snippets/product/type/configured/item-configure-button.liquid

```diff
@@ -0,0 +1,14 @@
+<a href="{{ product_url | append: '?resp_group=' | append: response_group }}">
+    <button class="btn btn-alt btn-primary configure-btn"
+    {% if unavailable or sold_out %} disabled {% endif %}
+    title="Configure product">
+    {% if unavailable %}
+    {{ 'products.product.unavailable' | t }}
+    {% elsif sold_out %}
+    {{ 'products.product.sold_out' | t }}
+    {% else %}
+    <img class="img-responsive configure-icon" src="{{ 'customize.svg' | asset_url }}" />
+    <span>Configure</span>
+    {% endif %}
+    </button>
+</a>
```

Comment:
May we use any formatting tool (for example, extension for VS Code) to format Liquid and indent `if-else` and similar blocks?

Reply:
done